build/common.gypi

```diff
@@ -25,6 +25,9 @@
       'disable_builtin_extensions%': '<(disable_builtin_extensions)',
     },
     'conditions': [
+      ['disable_web_audio==1', {
```

Comment:
Seems the variable enable_web_audio in gyp_xwalk could not affect this condition.